src/main/java/frc/robot/EnumLevel.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot;
+
+public enum EnumLevel {
+    // 0 in.
+    LOW_SCORE(0, 0),
+    // 34 in.
+    MEDIUM_SCORE(3.75, 78),
+    // 41 7/8 in.
+    HIGH_SCORE(19.6, 92),
+
+    FLOOR_PICKUP(0, 0),
```

Comment:
Is this number accurate?

Reply:
yes

Reply:
No, no it is not, just put that in there as we have not measured yet.